extide/gradle/manifest.mf

```diff
@@ -1,6 +1,6 @@
 Manifest-Version: 1.0
 AutoUpdate-Show-In-Client: false
-OpenIDE-Module: org.netbeans.modules.gradle/2
```

Comment:
I suggest to keep the removed methods around, just deprecated. Then you don't have to increment the release version and you can stick with `/2`.

Btw. the methods could even delegate to some provider to work the same way that did so far...

Reply:
Thanks! Great idea! I could return a <String, null> pair. That's supposed to be handled as a broken/invalid platform it would be sightly better than responding a straight null.